bot/exts/help_channels/_cog.py

```diff
@@ -580,7 +581,24 @@ async def notify_session_participants(self, message: discord.Message) -> None:
                 timestamp=message.created_at
             )
             embed.add_field(name="Conversation", value=f"[Jump to message]({message.jump_url})")
-            await message.author.send(embed=embed)
+
+            try:
+                await message.author.send(embed=embed)
+            except discord.errors.ForBidden:
+                log.trace(
+                    f"Failed to send helpdm message to {message.author.id}. DMs Closed/Blocked. "
+                    "Removing user from helpdm."
+                )
+                bot_commands_channel = self.bot.get_channel(Channels.bot_commands)
+                await _caches.help_dm.delete(message.author.id)
+                await bot_commands_channel.send(
+                    f"Hi, {message.author.mention} {constants.Emojis.cross_mark}. "
+                    f"Couldn't DM you helpdm message regarding {message.channel.mention} "
+                    "because your DMs are closed. helpdm has been automatically turned to off. "
+                    "to activate again type `!helpdm on`.",
```

Comment:
I think this message should be simplified.
```suggestion
                    f"{message.author.mention} {constants.Emojis.cross_mark} "
                    "To receive updates on help channels you're active in, unblock your DMs."
```

Reply:
Unblock may sound like you have the Python bot blocked. "please enable your DMs" or something of the sort may be a bit clearer.

Reply:
Agreed. I have updated the suggestion.